.gitignore

```diff
@@ -14,4 +14,5 @@ cygnus-twitter/bin
 .classpath
 .project
 .factorypath
-*/.settings
\ No newline at end of file
+*/.settings
+.*
```

Comment:
```suggestion
*/.settings
.*
```

as I understand that files like `.gitignore` itself should be included in the repo :)

Reply:
Fixed in 704e06e